scripts/hostmap-crtsh.nse

```diff
@@ -125,9 +118,7 @@ end
 
 local function write_file(filename, contents)
   local f, err = io.open(filename, "w")
-  if not f then
-    return f, err
-  end
+  if not f then return f, err end
```

Comment:
This change is not relevant. Focus your PR on the core enhancement, avoiding incidental changes.